src/main/avro/GuessService.avdl

```diff
@@ -0,0 +1,27 @@
+@version("1.0")
+@namespace("com.github.rakhmedovrs.avro")
+protocol GuessService {
+
+	enum RequestIntention {
+		MAKE_A_GUESS,
+		DROP_CONNECTION
+	}
+
+	enum GuessResult {
+		CORRECT,
+		INCORRECT,
+		UNDEFINED
+	}
+
+	record ClientGuessRequest {
+		int clientId;
+		RequestIntention requestIntention;
+		int guess;
+	}
+
+	record ServerGuessResponse {
```

Comment:
Is it possible that Server also may want to stop a connection by some reason?
In that case perhaps it is better to have different types (schemas) of messages, e.g. LoginMessage, DisconnectMessage, GuessMessage, GuessResponse .. and so on.